src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs

```diff
@@ -113,22 +119,47 @@ internal void SendNotification(BluetoothGattCharacteristic characteristic, strin
             GattUtil.ThrowForError(err, string.Format("Failed to send value changed notification for characteristic uuid {0}", characteristic.Uuid));
         }
 
-        internal Task<bool> SendIndicationAsync(BluetoothGattServer server, BluetoothGattCharacteristic characteristic, string clientAddress)
+        void SendIndicationCallback(int result, string clientAddress, IntPtr serverHandle, IntPtr characteristicHandle, bool completed, IntPtr userData)
         {
-            TaskCompletionSource<bool> tcs = new TaskCompletionSource<bool>();
-            Interop.Bluetooth.BtGattServerNotificationSentCallback cb = (result, address, serverHandle, characteristicHandle, completed, userData) =>
+            int requestId = (int)userData;
+            if (_sendIndicationTaskSource.ContainsKey(requestId))
             {
-                _notificationSent?.Invoke(characteristic, new NotificationSentEventArg(server, address, result, completed));
+                _notificationSent?.Invoke(characteristicHandle, new NotificationSentEventArg(_sendIndicationServer[requestId], clientAddress, result, completed));
```

Comment:
```suggestion
                _notificationSent?.Invoke(characteristicHandle, new NotificationSentEventArg(this, clientAddress, result, completed));
```

Reply:
Thank you!

Reply:
error occured. 'this' indicates GattServerImpl not GattServer.

Tizen.Network.Bluetooth/BluetoothGattImpl.cs(126): CS1503: Argument 1: cannot convert from 'Tizen.Network.Bluetooth.BluetoothGattServerImpl' to 'Tizen.Network.Bluetooth.BluetoothGattServer'

Reply:
Why sender is `characteristicHandle`?

Reply:
How about change like below?
``` c#
    public class NotificationSentEventArg : EventArgs
    {
        internal NotificationSentEventArg(BluetoothGattServer server, string clientAddress, int result, bool completed)
        {
            Result = result;
            ClientAddress = clientAddress;
            Server = server;
            Completed = completed;
        }
        public BluetoothGattServer Server { get; internal set;}
        public string ClientAddress { get; }
        public int Result { get; }
        public bool Completed { get; }
    }
...
       _notificationSent?.Invoke(characteristicHandle, new NotificationSentEventArg(null, clientAddress, result, completed));
...

        private BluetoothGattServer()
        {
            _impl = new BluetoothGattServerImpl();
            _impl._notificationSent += (s, e) => { 
                    e.Server = this;
                    NotificationSend?.Invoke(this, e); 
            };
        }

```